editions/tw5.com/tiddlers/wikitext/Formatting in WikiText.tid

```diff
@@ -1,19 +1,34 @@
 caption: Formatting
 created: 20131205155959399
-modified: 20220513120211686
+modified: 20220726132504220
 tags: WikiText
 title: Formatting in WikiText
 type: text/vnd.tiddlywiki
 
+<style>
+.no-code-border > code {
+  border: initial;
+  background-color: initial;
+}
+</style>
```

Comment:
(a) the core **never** uses local style blocks, nor do we recommend their use

(b) why aren't these styles in one of the docs stylesheets?

Reply:
b) Because I forgot to move them sorry.

Reply:
Ha! No problem